odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -506,6 +506,10 @@ class ComputeInverse(models.Model):
     foo = fields.Char()
     bar = fields.Char(compute='_compute_bar', inverse='_inverse_bar', store=True)
     baz = fields.Char()
+    child_ids = fields.One2many(
+        'test_new_api.compute.inverse', 'parent_id',
+        compute='_compute_child_ids', inverse='_inverse_child_ids', store=True)
+    parent_id = fields.Many2one('test_new_api.compute.inverse')
```

Comment:
Hierarchies are special.  I wouldn't use such a special case for testing a generic case.  Please add a plain stupid comodel for that one2many field.  Don't be lazy :wink: